src/Common/src/System/Globalization/FormatProvider.Number.cs

```diff
@@ -708,10 +705,9 @@ internal static unsafe string NumberToString(NumberBuffer number, char format, i
 
                 StringBuilder sb = new StringBuilder(MIN_SB_BUFFER_SIZE);
 
-                switch (format)
+                switch (char.ToUpper(format))
```

Comment:
Please revert this change.  The other changes thus far have just been about style, which is fine.  This one is a functional change, and while it's very likely the same performance, there is no validation for that, and the value of the change is low.